tracer/src/Datadog.Trace/Headers/Ip/RequestIpExtractor.cs

```diff
@@ -0,0 +1,71 @@
+// <copyright file="RequestIpExtractor.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.Headers.Ip
+{
+    internal static class RequestIpExtractor
+    {
+        private static readonly IReadOnlyList<string> IpHeaders = new[] { "x-forwarded-for", "x-real-ip", "client-ip", "x-forwarded", "x-cluster-client-ip", "forwarded-for", "forwarded", "via", "true-client-ip" };
+        internal static readonly IReadOnlyList<string> MainHeaders = new[] { "user-agent", "referer" };
```

Comment:
Is it not used or is it used in files I have reviewed and closed? (sorry I don't have the courage to reopen them)